src/args.js

```diff
@@ -58,20 +58,20 @@ export default yargs
       describe: 'The JSON configuration file.',
       type: 'string'
     },
-    strip: {
-      alias: 's',
-      describe: 'Strip the identifier field for each object type.',
-      type: 'boolean',
-      default: false
-    },
     secret: {
       alias: 'x',
       describe: 'The client secret, this allows you to encrypt the secret in your build configuration instead of storing it in a config file',
       type: 'string'
+    },
```

Comment:
Probably should remove this. If we're going to use `AUTH0_EXPORT_IDENTIFIERS: true`

Reply:
And we decided to keep `export_ids` param. If set, it'll override config value.